README.md

```diff
@@ -136,6 +136,8 @@ export default createHandler(User);
 
 |                         | Description                                 |
 | ----------------------- | ------------------------------------------- |
+| `@Req()`                | Gets the request object.                    |
+| `@Res()`                | Gets the response object.                   |
```

Comment:
When `@Res()` is injected, the developer is responsible for handling the response. So, we should add something like:

*When using `@Res()`, you are in charge of sending the response to the client. Therefore, `return` statement won't be handled by this package and the response won't be served.*